Source/DevAddin/InitializeHome.jsl

```diff
@@ -6,7 +6,7 @@ Names Default To Here(0);
 
 ut addin scratch = Convert File Path("$ADDIN_HOME(com.jmp.jslhamcrestdev)");
 ut addin home = Try(
-	Load Text File(ut addin scratch || "/DevConfiguration.txt"),
+	Load Text File(ut addin scratch || "/HomeConfiguration.txt"),
```

Comment:
Remove extra `/`. I am not sure if it would work with double slashes on both hosts.